webapp-django/crashstats/status/context_processors.py

```diff
@@ -0,0 +1,18 @@
+from crashstats.status.models import StatusMessage
+
+
+def status_message(request):
+    statuses = (
+        StatusMessage.objects.filter(enabled=True).order_by('-created_at')
+    )
+
+    if not len(statuses):
+        return {}
+
+    status = statuses[0]
+    return {
+        'status_message': {
+            'text': status.message,
+            'severity': status.severity,
+        },
+    }
```

Comment:
I think this whole thing should be made into a list. E.g.
```python
def status_message(request):
    qs = StatusMessage.objects.filter(enabled=True).order_by('-created_at')
    messages = []
    for status in qs:
        messages.append({ ... })
    return {'status_messages': messages}
```
That takes care of making it possible to have more than one at the same time.